Quantum-Control-Applications/Superconducting/Single Fixed Transmon/rb.py

```diff
@@ -125,67 +126,84 @@ def play_sequence(sequence_list, depth):
     Q = declare(fixed)
     state = declare(bool)
     state_st = declare_stream()
+    depth_target = declare(int)
 
     with for_(m, 0, m < num_of_sequences, m + 1):
         sequence_list, inv_gate_list = generate_sequence()
 
+        assign(depth_target, 0)
+
         with for_(depth, 1, depth <= max_circuit_depth, depth + delta_depth):
-            with for_(n, 0, n < n_avg, n + 1):
-                # Replacing the last gate in the sequence with the sequence's inverse gate
-                # The original gate is saved in 'saved_gate' and is being restored at the end
-                assign(saved_gate, sequence_list[depth])
-                assign(sequence_list[depth], inv_gate_list[depth - 1])
+            # Replacing the last gate in the sequence with the sequence's inverse gate
+            # The original gate is saved in 'saved_gate' and is being restored at the end
+            assign(saved_gate, sequence_list[depth])
+            assign(sequence_list[depth], inv_gate_list[depth - 1])
```

Comment:
This is actually what prevents us from setting delta_depth > 1 right ?
Maybe we can hardcode delta_depth=1 then, what do you think?